src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -878,7 +878,8 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | RETURN_LITERAL (WS | NEWLINE)* description?
 
-      | SEE_LITERAL (WS | NEWLINE)* reference? (STRING | htmlElement)* (WS | NEWLINE)* description?
+      | SEE_LITERAL (WS | NEWLINE)* reference? ((WS | NEWLINE)+ STRING | htmlElement)*
+      (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
Ok, but you can yourself that regexp is not consistent for all it's parts, that is strange and smells.
What about `SEE_LITERAL (WS | NEWLINE)+ reference? (WS | NEWLINE)+ (STRING | htmlElement)* (WS | NEWLINE)+ ( description)?` ?